tests/test_js_utils.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from tests.conftest import BaseGraphCase
+from tests.conftest import GraphCaseFactory
+
+
+@pytest.fixture
+def graph(graph_cases: GraphCaseFactory) -> BaseGraphCase:
+    return graph_cases('empty')
+
+
+def test_set_style_key(graph: BaseGraphCase) -> None:
+    assert graph.eval_js_function('graphs.utils.setStyleKey', 'style', 'bar', 5) == 'style;bar=5'
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'style;bar=7', 'bar', 5) == 'style;bar=5'
+    )
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'foobar=3;bar=7', 'bar', 5)
+        == 'foobar=3;bar=5'
+    )
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'foobar=3', 'bar', 5) == 'foobar=3;bar=5'
+    )
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'foobar=3;fizzbar=7', 'bar', 5)
+        == 'foobar=3;fizzbar=7;bar=5'
+    )
+
+
+def test_remove_style_key(graph: BaseGraphCase) -> None:
+    assert graph.eval_js_function('graphs.utils.removeStyleKey', 'style;bar=7', 'bar') == 'style'
```

Comment:
Please add another test case:

```suggestion
    assert graph.eval_js_function('graphs.utils.removeStyleKey', '', 'bar') == ''
```